core/client/fs/src/main/java/alluxio/client/file/FileInStream.java

```diff
@@ -163,8 +165,10 @@ public int read(byte[] b, int off, int len) throws IOException {
         lastException = null;
       } catch (UnavailableException | ConnectException | DeadlineExceededException e) {
         lastException = e;
-        handleRetryableException(mBlockInStream, e);
```

Comment:
put line 156 `updateStream();` into this `try` block too?

Reply:
I have put a new submit to fix it.